Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml

```diff
@@ -1585,7 +1533,7 @@ tasks:
             args:
               key:
                 value:
-                  simple: QRadar.Search.Result.events
+                  simple: QRadar.SearchEvents.Events
```

Comment:
was the output of the previous command changed?

Reply:
Yes